frontends/web/src/routes/account/manage/components/coin-dropdown.tsx

```diff
@@ -133,9 +42,10 @@ function CoinDropDown({
                     disabled: true,
                     value: 'choose',
                 },
-                ...(supportedCoins).map(item => ({
-                    value: item,
-                    text: COIN_AND_ACCOUNT_CODES[item].name,
+                ...(supportedCoins).map(({ coinCode, name, canAddAccount}) => ({
+                    value: coinCode,
+                    text: name,
+                    disabled: !canAddAccount,
```

Comment:
:+1: discussed with @x1ddos . We should probably add a sentence like 'Up to five accounts per coin are supported' (which is what we settled on for now, could change later).

Reply:
When would you expect to see this message?

once you try to add the 6th account, or already at the beginning of the workflow, or just in the guide?

cc @jadzeidan wdyt?

Reply:
can be done later, lets merge this.

Reply:
If the coin is disabled like in this PR, then you can't even add the account, so it would have to be before.

No strong opinions anyway.

@jadzeidan @thisconnect fyi, the plan for now is not go into the gaplimit-for-accounts business and just limit to the accounts per coin and bitbox to e.g. 5, which is a loooot simpler conceptually and in code (e.g. we don't need to do any scanning to decide if the user can add an account). We can still switch to gap limits later if 5 accounts is not good enough.